experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -162,6 +164,41 @@ func checkInclusionProof(logEntryAnon *models.LogEntryAnon, registry strfmt.Regi
 	return logEntryAnon.Validate(strfmt.Default)
 }
 
+func verifySignedEntryTimestamp(logEntryAnon *models.LogEntryAnon, rekorPublicKeyBytes []byte) error {
```

Comment:
I think it is worth adding a unit test (e.g., `test_ verifySignedEntryTimestamp`), just for this function.

Reply:
fixed